SonarQube.MSBuild.Tasks/GetAnalyzerSettings.cs

```diff
@@ -57,25 +63,38 @@ public override bool Execute()
 
             AnalysisConfig config = TaskUtilities.TryGetConfig(this.AnalysisConfigDir, new MSBuildLoggerAdapter(this.Log));
             
-            if (config != null)
+            if (config != null && Language != null)
```

Comment:
It's consisted with config; if it's not defined, means that we shouldn't run at all. We can't use required, otherwise it will fail with an error if Language is not defined in the target.